exec/src/klio_exec/commands/run.py

```diff
@@ -540,6 +557,21 @@ def _setup_pipeline(self, pipeline):
 
         out_pcol = run_callable(to_process, self.config)
 
+        # TODO: update me to `var.KlioRunner.DIRECT_GKE_RUNNER` once
+        #       direct_on_gke_runner_clean is merged
+        if self.config.pipeline_options.runner == "DirectGKERunner":
+            if to_pass_thru:
+                to_ack_input = (
+                    out_pcol,
+                    to_pass_thru,
+                ) | "Flatten to Ack Input Messages" >> beam.Flatten()
+            else:
+                to_ack_input = out_pcol
+
+            _ = to_ack_input | "Ack Input Messages" >> beam.ParDo(
+                helpers.KlioAckInputMessage()
```

Comment:
Do we need to be acking messages ourselves if not using our custom direct runner?

It looks like when I run with regular direct runner, I get a `WARNING:klio.gke_direct_runner.message_manager:Unable to acknowledge test: Not found.` when it processes a message.  Haven't tried on dataflow yet.

Reply:
Hmm I'm confused, how does this line get hit with the `if` clause on 562?

Reply:
ah nevermind I misread the line, and I think the warning I'm seeing was due to not having our other branch merged when I was testing.  I'll try again now that everything is merged together anyway